prac_07/project_management.py

```diff
@@ -0,0 +1,129 @@
+"""
+CP1404 project_management.py
+
+Estimate: 3 hours
+actual:
+"""
+from project import Project
+import datetime
+from operator import attrgetter
+
+MENU = """(L)oad projects
+(S)ave projects
+(D)isplay projects
+(F)ilter projects by date
+(A)dd new project
+(U)pdate project
+(Q)uit"""
```

Comment:
There was an option to load projects in the sample output if I remember correctly